Consistent labeling of kyma module: API-Gateway (#812)
kyma-project/api-gateway: docs/user/00-10-overview-api-gateway-controller.md

| `Processing` | Controller is reconciling resources.     |
|  `Deleting`  | Controller is deleting resources.        |
|   `Error`    | An error occurred during reconciliation. |
|  `Warning`   | Controller is misconfigured.             |

> ## Labeling resources

Reviewer: ```suggestion
## Labeling Resources
```

> Based on the decision taken for [Consistent labeling of Kyma modules](https://github.com/kyma-project/community/issues/864). APIGateway operator resources are labeled with the common kubernetes labels:

Reviewer: ```suggestion
In accordance with the decision [Consistent Labeling of Kyma Modules](https://github.com/kyma-project/community/issues/864), the APIGateway Operator resources use the standard Kubernetes labels:

```

```yaml
kyma-project.io/module: api-gateway
app.kubernetes.io/name: api-gateway-operator
app.kubernetes.io/instance: api-gateway-operator-default
app.kubernetes.io/version: "x.x.x"
app.kubernetes.io/component: operator
app.kubernetes.io/part-of: api-gateway
```

For all other resource like the external `ory-oathkeeper` component and its respective resources we only apply the Kyma module label:

```yaml
kyma-project.io/module: api-gateway
```

> Getting all resources created by the API Gateway module is possible by running:

Reviewer: ```suggestion
Run this command to get all resources created by the API Gateway module:
```
